client/Client.cpp

```
@@ -313,6 +313,22 @@ void Client::sendPosition() {
   }
 }
 
+void Client::sendInput(MovementInputType input) {
```

Clang (14.0.6)

[warning] method 'sendInput' can be made static

(readability-convert-member-functions-to-static)